gradle/libs.versions.toml

```diff
@@ -77,6 +80,9 @@ ksp-gradlePlugin = { group = "com.google.devtools.ksp", name = "com.google.devto
 room-gradlePlugin = { group = "androidx.room", name = "room-gradle-plugin", version.ref = "room" }
 google-play-publish-gradlePlugin = { group = "com.github.triplet.gradle", name = "play-publisher", version.ref = "google-play-publish" }
 advanced-gradle-build-version-gradlePlugin = { group = "me.moallemi.gradle", name = "advanced-build-version", version.ref = "advanced-gradle-build-version" }
+compose-foundation = { module = "androidx.compose.foundation:foundation", version.ref = "compose-foundation" }
+androidx-tv-material = { group = "androidx.tv", name = "tv-material", version.ref = "tv-material" }
+tv-compose-navigation = { module = "androidx.navigation:navigation-compose", version.ref = "tv-compose-navigation" }
```

Comment:
Yeah, you're right, I didn't want to upgrade that dependency because most of the features depended on that and I thought maybe this approach is better. 
I'll fix it

Reply:
Again: if updating it to latest navigation version breaks the mobile app functionality, continue with your approach.